client/src/utils/ReviewStar.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function ReviewStar({ currentReviewsMetadata }) {
+  function calAvgRating(MetadataRating) {
+    let total = 0;
+    let number = 0;
+    for (let i = 0; i < 5; i++) {
+      if (MetadataRating[i + 1] !== undefined) {
+        total = total + (i + 1) * (MetadataRating[i + 1]);
+        number = number + (parseInt(MetadataRating[i + 1]));
+      }
+    };
+    let avg = total / number;
+    const left = avg % 0.25;
```

Comment:
The requirements say that the quarter of star should be rounded.  Right now 2.24 maps to 2.0, and should map to 2.25